books/free-programming-books-id.md

```diff
@@ -159,3 +163,11 @@
 ### Vue
 
 * [Dokumentasi Vue Bahasa Indonesia](https://github.com/vuejs-id/docs)
+
+### Laravel
+
+* [Belajar Laravel Untuk Pemula](https://gilacoding.com/upload/file/Belajar%20Laravel%20Untuk%20Pemula.pdf) Type : PDF Author : Dadan Hamdani
+
+### Yii 
+
+* [Menjelajahi Yii Framework](https://gilacoding.com/upload/file/menjelajahyiiframework.pdf) Type : PDF Author : Dida	Nurwanda
```

Comment:
Please:
- move it to PHP section
- fix resource format
- Give right creators attribution: <del>Dida Nurwanda</del> -> Sabit Huraira

```markdown
* [Menjelajahi Yii Framework](https://gilacoding.com/upload/file/menjelajahyiiframework.pdf) - Sabit Huraira (PDF)
```